src/verilator/VerilatorDinoCPU.cc

```diff
@@ -0,0 +1,79 @@
+#include "VerilatorDinoCPU.hh"
+#include "base/logging.hh"
+#include "sim/sim_exit.hh"
+
+VerilatorDinoCPU::VerilatorDinoCPU(VerilatorDinoCPUParams *params) :
+    SimObject(params),
+    event([this]{updateCycle();}, params -> name),
+    latency(params -> latency),
```

Comment:
I've never seen spaces around the `->` operator before. I don't think you should do that. It would be like spaces around the `.` operator. `cpu . memory` doesn't look right :)